app/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'atlas-web-app.java-conventions'
```

Comment:
As far as I know, the build logic in `atlas-web-app.java-conventions` is not shared by any other projects in this repo. Is it necessary to have a seperate gradle build file here?

Reply:
I got the answer in our meeting. Since I have not pulled the latest submodule, I don't have this plugin in web-core project.